libdnf.spec

```diff
@@ -154,6 +155,7 @@ popd
 %{_libdir}/pkgconfig/%{name}.pc
 %{_includedir}/%{name}/
 %{_datadir}/gir-1.0/Dnf-*.gir
+%{_datadir}/vala/vapi/
```

Comment:
Please don't. We have a convention in Fedora to put vala bindings in the main -devel package without depending on the vala package. They are just tiny files that shouldn't affect anyone.

Ahh right, sorry @ignatenkobrain, I missed that directory. So either

%dir %{_datadir}/vala
%dir %{_datadir}/vala/vapi
%{_datadir}/vala/vapi/libdnf-*.vapi

or simply "%{_datadir}/vala/" should be equivalent.